meson.build

```diff
@@ -118,6 +118,7 @@ if get_option('ipc')
 
     nanoarrow_ipc_lib = library(
```

Comment:
Does this need the zstd dependency? Looks like it is in the wrapdb, so can just `meson wrap install zstd` to get the wrap file created then

```
zstd_dep = dependency('zstd')
nanoarrow_ipc_lib = library(
    ...,
    dependencies: [zstd_dep, ...]
)
````